app/database/models.py

```diff
@@ -61,8 +61,8 @@ class Event(Base):
     color = Column(String, nullable=True)
     availability = Column(Boolean, default=True, nullable=False)
     invitees = Column(String)
+    is_public = Column(Boolean, default=False)
```

Comment:
Event doesn't have "friends" option?

Reply:
the meaning of friends is different- public event is meant to allow any user in the platform to join the event, just like the public events on facebook. so it means the owner's friends don't matter in this feature :)

Reply:
Can you please explain what is the meaning of public/private event? is it like busy/free?
Thanks

Reply:
a pubic event is event anyone can join. it means you don't need to be invited directly from the owner, you can join the event by yourself by clicking a button on event page